lldb-eval/parser_test.cc

```diff
@@ -28,30 +28,30 @@
 
 namespace {
```

Comment:
So I had a look around and looks like the gdb people have fixed this: https://sourceware.org/bugzilla/show_bug.cgi?id=16874

However, I also spotted e.g. https://youtrack.jetbrains.com/issue/CPP-18948 (which is related to CLion), so even with the gdb fix in mind, I would still go with `static` instead of an anonymous namespace.